control-plane/pkg/reconciler/channel/controller.go

```diff
@@ -89,10 +90,12 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *conf
 		)
 	}
 
+	caCerts, err := reconciler.getCaCerts()
```

Comment:
Is it fine

Reply:
We still need to check the error here and see if the error makes sense given the TLS flags. See here: https://github.com/knative-sandbox/eventing-kafka-broker/blob/32dfed583ceb02c4fa1581b3bffab07d6ec0e7d6/control-plane/pkg/reconciler/broker/controller.go#L100-L103